python/tests/test_model.py

```diff
@@ -57,4 +57,10 @@ def test_bad_ground_to_image(model):
     gnd_coord = csmapi.EcefCoord(-1, -1, 0)
     with pytest.warns(Warning) as w:
         img = model.groundToImage(gnd_coord, 0)
-        assert len(w) == 1
\ No newline at end of file
+        assert len(w) == 1
+
+def test_ellipsoid_is_settable(model):
+    assert isinstance(csmapi.SettableEllipsoid.getEllipsoid(model), csmapi.Ellipsoid)
+    print(dir(model))
+    print(model.getEllipsoid(), type(model.getEllipsoid()))
```

Comment:
I don't think we're expected to make this call. The ellipsoid should be accessed via the static `SettableEllipsoid::getEllipsoid` function as in the first assert.

Reply:
Shouldn't we be able to do:

```python
model = my_model
ell = model.getEllipsoid()
```